front/src/components/login/LoginForm.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { LoginButton, StyledCol, ForgotPassword, ErrorText} from './styledLoginForm';
+import { Form, Input, Col, Typography, Row } from 'antd';
+import 'antd/dist/antd.css';
+
+const { Title } = Typography;
+const layout = { 
+    labelCol: { span: 8, },
+    wrapperCol: { span: 16, },
+};
+
+export class LoginForm extends Component {
+
+    async validatePassword(_, value) {
```

Comment:
Don't write async methods inside the component

Reply:
done